roles/sahara/defaults/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+sahara:
+  enabled: True
+  juno_url: http://tarballs.openstack.org/sahara/sahara-stable-juno.tar.gz
+  dashboard_url: http://tarballs.openstack.org/sahara-dashboard/sahara-dashboard-master.tar.gz
```

Comment:
This should leverage our existing routine to fetch source via git, instead of a tarball.